portstyle.css

```diff
@@ -91,112 +94,105 @@ body {
 	align-self: center;
 }
```

Comment:
This image is going to resize based on it parent meaning it's gonna get pretty tiny on mobile, consider removing the width property and adding min-width property

Reply:
I have set it to width: 100% and have changed the media query so that is resizes for mobile nd desktop.